### ISSUE TITLE
Connection problem with twitter REST API

### ISSUE THREAD

**Issue by johnpiers** [orakaro/rainbowstream#342]
Everything was working fine until a few hours ago now I'm getting this: Is it only me or anyone one else?  Or is this the end of Rainbowstream. That would be mighty sad.
Thanks, I would be grateful if anyone would let me know. Cheers 

"We have connection problem with twitter REST API right now :(
You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level."

 You can learn more here: https://developer.twitter.com/en/portal/product

**Comment by g33k247** [comment]
Also seeing this today.

**Comment by awful-piper** [comment]
same here,started yesterday.

**Comment by johnpiers** [comment]
Ok, I suppose I was hoping that it was just me and not what I already knew. Is this 'fixable' or is it really the end?   😫 😫

**Comment by eqhmcow** [comment]
not fixable w/o paying the thousands of dollars a month to twitter for API access, if I understand correctly :(

we'll need to write a new app that uses bluesky instead! :)

**Comment by awful-piper** [comment]
i always thought rainbowstream would make a great rss reader. if rainbowstream does not return.. thanks to the devs for making and maintaining rainbowstream for so long. i've used it for years, everyone who sees it on my desktop asks about it. 
![FtwO3o4WIAAEqGW](https://github.com/orakaro/rainbowstream/assets/117608665/9d8cd052-9834-4a05-b3d4-6df7ebe857ce)

**Comment by umop3plsdn** [comment]
I realize rainbowstream is gone forever. I just want to thank you for it's creation in the first place and it was a pleasure to even use it for as long as i had <3. This app was hella underrated and deserved way more attention

**Comment by johnpiers** [comment]
> I realize rainbowstream is gone forever. I just want to thank you for it's creation in the first place and it was a pleasure to even use it for as long as i had <3. This app was hella underrated and deserved way more attention

Concur with your post most wholeheartedly (albeit sadly). The first thing I used to do on booting my linux box, was to open a terminal and fire up Rainbowstream.  I use Arch btw! 😋 😋

**Comment by dnet890** [comment]
yeah, it's gone and I got the same issue. I wish we can port this client to mastodon instead

**Comment by johnpiers** [comment]
> yeah, it's gone and I got the same issue. I wish we can port this client to mastodon instead

A cli client for Mastodon that can do pretty much most of what you need on Mastodon is called "Tootstream"
